Source/Kernel/Concepts/Configuration/Observers.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Cratis. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cratis.Chronicle.Concepts.Configuration;
+
+/// <summary>
+/// Represents the configuration for observers.
+/// </summary>
+public class Observers
+{
+    /// <summary>
+    /// Gets the timeout in seconds for observer calling its subscriber.
+    /// </summary>
+    public int SubscriberTimeoutInSeconds { get; init; } = 5;
```

Comment:
Perhaps drop the `InSeconds` in name and leave that it for XML comments and documentation.
(see below commend)